src/SFA.DAS.FAT.Web/Views/CourseProviders/CourseProviders.cshtml

```diff
@@ -65,7 +64,7 @@
                     {
                         <div class="govuk-grid-column-two-thirds govuk-!-padding-0">
                             <p class="govuk-body govuk-!-text-align-right">
-                                <label class="govuk-label" for="sort">
+                                    <label class="govuk-label" for="course-providers-order-form">
```

Comment:
for should be `course-providers-orderby` which is the id of the drop down

Reply:
my bad, I thought it was for the form

Reply:
fixed